core/modules/startup/keyboard.js

```diff
@@ -0,0 +1,51 @@
+/*\
+title: $:/core/modules/startup/keyboard.js
+type: application/javascript
+module-type: startup
+
+Keyboard shortcut handling
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+// Export name and synchronous status
+exports.name = "global-shortcuts";
+exports.platforms = ["browser"];
+exports.after = ["startup"];
+exports.synchronous = true;
+
+exports.startup = function() {
+	if($tw.browser) {
+		var man = $tw.keyboardManager;
+		// Get existing shortcut-tiddlers
+		man.shortcutTiddlers = man.getShortcutTiddlerList(),
+		man.shortcutKeysList = [], // Stores the shortcut-key descriptors
+		man.shortcutActionList = [], // Stores the corresponding action strings
+		man.shortcutParsedList = []; // Stores the parsed key descriptors
```

Comment:
Wouldn't it make more sense to do the above initialisation within the constructor?